src/pages/Video.js

```diff
@@ -0,0 +1,89 @@
+import { useParams } from "react-router-dom";
+import { ReactComponent as ThumbsUpIcon } from "../assets/thumbs-up.svg";
+import { ReactComponent as ThumbsDownIcon } from "../assets/thumbs-down.svg";
+import Navbar from "../components/Navbar";
+import { useEffect, useState } from "react";
+import Spinner from "../components/Spinner";
+import { apiBaseURL } from "../constants";
+import VideoCard from "../components/VideoCard";
+
+export default function Video() {
+  const { id } = useParams();
+
+  const [video, setVideo] = useState(null);
+  const [videoLoading, setLoading] = useState(true);
+  const [recommendedVideos, setRecommendedVideos] = useState([]);
+  const [recommendedVideosLoading, setRecommendedVideosLoading] =
+    useState(true);
+
+  useEffect(() => {
```

Comment:
All this code is super readable i think. We'll make some time to explain the useEffect & useState